libs/Task.js

```diff
@@ -34,7 +34,7 @@ let Directories = require('./Directories');
 let statusCodes = require('./statusCodes');
 
 module.exports = class Task{
-	constructor(uuid, name, done, options = []){
+	constructor(uuid, name, done, options = [], webhook){
```

Comment:
Webhook mechanism is good, but the webhook will be lost in case the server is reset (to reproduce):

1. Start a task with a webhook
2. CTRL+C the process
3. Restart the process
4. The task is loaded from `tasks.json`, but webhook information is gone.

See `Task.serialize` and `Task.CreateFromSerialized`. Could we add the webhook property there?